drake_ros/drake_ros/systems/__init__.py

```diff
@@ -1,10 +1,14 @@
 from .joint_state_publisher import JointStatePublisher
+from .moveable_joints import MoveableJoints
```

Comment:
Er, is the spelling `Moveable` or `Movable`?

Reply:
From https://github.com/RobotLocomotion/drake/commit/40876137bf2931f0a15641fc0f4a57fbd9c69c12:
```
$ grep -rnI 'movable' | wc -l
22
$ grep -rnI 'moveable' | wc -l
12
```

Reply:
Moveable -> Movable in 357ad75